packages/config/config/devices/0x001d/templates/config_template.json

```diff
@@ -0,0 +1,163 @@
+{
+	"factory_default": {
+		"label": "Factory Default",
+		"$import": "~/templates/master_template.json#base_enable_disable"
+	},
+	"auto_buzz": {
+		"label": "Auto Buzzer",
+		"valueSize": 4,
+		"defaultValue": 1,
+		"readOnly": true,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Disable",
+				"value": 0
+			},
+			{
+				"label": "Enable",
+				"value": 1
+			}
+		]
+	},
+	"auto_lock": {
+		"label": "Auto Relock",
+		"valueSize": 4,
+		"defaultValue": 0,
+		"readOnly": true,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Disable",
+				"value": 0
+			},
+			{
+				"label": "Enable",
+				"value": 1
+			}
+		]
+	},
+	"led_status": {
+		"label": "Lock LED Status",
+		"description": "Door lock status LED blinks every 6 seconds.",
+		"valueSize": 4,
+		"defaultValue": 1,
+		"readOnly": true,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Disable",
+				"value": 0
+			},
+			{
+				"label": "Enable",
+				"value": 1
+			}
+		]
+	},
+	"locator_led": {
+		"label": "Locator LED Status",
+		"valueSize": 1,
+		"defaultValue": 255,
+		"unsigned": true,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "LED always off",
+				"value": 0
+			},
+			{
+				"label": "LED on when switch is on",
+				"value": 254
+			},
+			{
+				"label": "LED on when switch is off",
+				"value": 255
+			}
+		]
+	},
+	"load_type": {
+		"label": "Load Type",
+		"valueSize": 1,
+		"defaultValue": 0,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Incandescent",
+				"value": 0
+			},
+			{
+				"label": "LED",
+				"value": 1
+			},
+			{
+				"label": "CFL",
+				"value": 2
+			}
+		]
+	},
+	"led_timeout": {
+		"label": "LED Dim Level Indicator Timeout",
+		"description": "How long the level indicators should stay illuminated after the dimming level is changed",
+		"valueSize": 1,
+		"unit": "seconds",
+		"minValue": 0,
+		"maxValue": 255,
+		"defaultValue": 3,
+		"unsigned": true,
+		"options": [
+			{
+				"label": "Always Off",
+				"value": 0
+			},
+			{
+				"label": "Always On",
+				"value": 255
+			}
+		]
+	},
+	"preset_level": {
+		"label": "Preset Light Level",
+		"valueSize": 1,
+		"unit": "%",
+		"minValue": 0,
+		"maxValue": 100,
```

Comment:
Is this actually 0-100%? Levels usually go from (0) 1-99 in Z-Wave

Reply:
I have checked multiple Leviton device documents and they do indeed accept 0-100 as values.